openverse_catalog/dags/util/loader/sql.py

```diff
@@ -40,94 +68,40 @@ def create_loading_table(
     """
     Create intermediary table and indices if they do not exist
     """
+
+    def create_index(column, btree_column=None):
```

Comment:
I'm not a huge fan of having this scoped function reference variables that aren't defined yet - would it be possible to move this down below line 99?

Reply:
Done 👍